### ISSUE TITLE
Remove timestamps from contracts pt 2

### ISSUE THREAD

**Issue by onewhiskeypls** [DA0-DA0/dao-contracts#496]
Part 2 of #485 - we'll need to review the cw-proposal-multiple contract to ensure `last_updated` and `created` timestamps are removed from code.  A mild attempt to remove it was made in the original PR tied to #485 , but that version was in a rough state, so I made a note to review it once that contract is merged in

**Comment by JakeHartnell** [comment]
Closing in favor of #485
